smart-contracts/contracts/Unlock.sol

```diff
@@ -242,22 +246,42 @@ contract Unlock is
           // Get the value of 1 UDT (w/ 18 decimals) in ETH
           uint udtPrice = udtOracle.updateAndConsult(udt, 10 ** 18, weth);
 
-          // tokensToMint is either == to the gas cost times 1.25 to cover the 20% dev cut
-          uint tokensToMint = (estimatedGasForPurchase * tx.gasprice).mul(125 * 10 ** 18) / 100 / udtPrice;
-          // or tokensToMint is capped by percent growth
-          uint maxTokens = IMintableERC20(udt).totalSupply().mul(valueInETH) / 2 / grossNetworkProduct;
-          if(tokensToMint > maxTokens)
+          // tokensToDistribute is either == to the gas cost times 1.25 to cover the 20% dev cut
+          uint tokensToDistribute = (estimatedGasForPurchase * tx.gasprice).mul(125 * 10 ** 18) / 100 / udtPrice;
+
+          // or tokensToDistribute is capped by network GDP growth
+          uint maxTokens = 0;
+          if (chainId > 1)
```

Comment:
Exactly.

Reply:
I was thinking of using a flag like `isMainNet` but thought it would be more flexible to use the chainId as we may in the future want to have different behaviors based on the chain id (not jist mainnet vs non mainnet)